<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FIC-I7KL93" failures="0" tests="4" name="com.evaa.chatbot.BookAppointmentSuccessfullyTest" time="119.812" errors="1" timestamp="2025-06-11T15:39:03 IST" skipped="2">
  <testcase classname="com.evaa.chatbot.BookAppointmentSuccessfullyTest" name="Disable_ensurance_required_from_admin" time="22.393"/>
  <system-out/>
  <testcase classname="com.evaa.chatbot.BookAppointmentSuccessfullyTest" name="Test_book_appointment_without_insurance" time="89.072">
    <skipped/>
  </testcase> <!-- Test_book_appointment_without_insurance -->
  <system-out/>
  <testcase classname="com.evaa.chatbot.BookAppointmentSuccessfullyTest" name="Test_book_appointment_without_insurance" time="8.347">
    <error message="element click intercepted: Element &lt;button id=&quot;chat_submit&quot; class=&quot;btn p-2 pt-3&quot; type=&quot;submit&quot; name=&quot;submit&quot; data-kt-element=&quot;send&quot; disabled=&quot;disabled&quot;&gt;...&lt;/button&gt; is not clickable at point (398, 721). Other element would receive the click: &lt;div class=&quot;col-1 p-0&quot; style=&quot;display: flex; flex-direction: row;justify-content: flex-end;padding-right: 0;&quot;&gt;...&lt;/div&gt;
  (Session info: chrome=137.0.7151.69)
Build info: version: &#039;4.1.4&#039;, revision: &#039;535d840ee2&#039;
System info: host: &#039;FIC-I7KL93&#039;, ip: &#039;172.20.10.13&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.6&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [828fe4cc5240614b5daaa13bef6931ca, clickElement {id=f.A9C8C3DB0E86C09AFA931CE3C650E8CC.d.7F9ED6F0A4E122A680E3DDB3641288EC.e.7}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.70 (dfa4dc56b2ab..., userDataDir: C:\Users\Satish_G\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:55818}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:55818/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (828fe4cc5240614b5daaa13bef6931ca)] -&gt; xpath: //button[@id=&#039;chat_submit&#039;]]
Session ID: 828fe4cc5240614b5daaa13bef6931ca" type="org.openqa.selenium.ElementClickInterceptedException">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <button id="chat_submit" class="btn p-2 pt-3" type="submit" name="submit" data-kt-element="send" disabled="disabled">...</button> is not clickable at point (398, 721). Other element would receive the click: <div class="col-1 p-0" style="display: flex; flex-direction: row;justify-content: flex-end;padding-right: 0;">...</div>
  (Session info: chrome=137.0.7151.69)
Build info: version: '4.1.4', revision: '535d840ee2'
System info: host: 'FIC-I7KL93', ip: '172.20.10.13', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.6'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [828fe4cc5240614b5daaa13bef6931ca, clickElement {id=f.A9C8C3DB0E86C09AFA931CE3C650E8CC.d.7F9ED6F0A4E122A680E3DDB3641288EC.e.7}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.70 (dfa4dc56b2ab..., userDataDir: C:\Users\Satish_G\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:55818}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:55818/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (828fe4cc5240614b5daaa13bef6931ca)] -> xpath: //button[@id='chat_submit']]
Session ID: 828fe4cc5240614b5daaa13bef6931ca
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:567)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:251)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:77)
at com.evaa.chatbot.BookAppointmentSuccessfullyTest.Test_book_appointment_without_insurance(BookAppointmentSuccessfullyTest.java:86)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- Test_book_appointment_without_insurance -->
  <system-out/>
  <testcase classname="com.evaa.chatbot.BookAppointmentSuccessfullyTest" name="verify_detail_on_maximeyes_site">
    <skipped/>
  </testcase> <!-- verify_detail_on_maximeyes_site -->
  <system-out/>
</testsuite> <!-- com.evaa.chatbot.BookAppointmentSuccessfullyTest -->
